app/uk/gov/hmrc/api/testlogin/services/LoginService.scala

```diff
@@ -16,20 +16,19 @@
 
 package uk.gov.hmrc.api.testlogin.services
 
+import java.time.{Clock, Instant}
 import java.util.UUID
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
-import org.joda.time.DateTime
-
 import play.api.mvc.Session
 import uk.gov.hmrc.api.testlogin.connectors.ApiPlatformTestUserConnector
 import uk.gov.hmrc.api.testlogin.models.{AuthenticatedSession, LoginRequest}
 import uk.gov.hmrc.http.SessionKeys._
 import uk.gov.hmrc.http.{HeaderCarrier, SessionId}
 
 @Singleton
-class LoginService @Inject() (apiPlatformTestUserConnector: ApiPlatformTestUserConnector) {
+class LoginService @Inject() (apiPlatformTestUserConnector: ApiPlatformTestUserConnector, clock: Clock) {
```

Comment:
possibly worth using the ClockNow trait from the common lib.

Reply:
Done